### ISSUE TITLE
Translation of bulleted or enumerated lists is not working.

### ISSUE THREAD

**Issue by Legi0nP** [smark-d/epub-translator#6]
Translation of bulleted or enumerated lists is not working.

**Comment by smark-d** [comment]
Can you provide the original file? Or a fragment of the original file (unzip the epub file and send me some xml/htm files)?

**Comment by Legi0nP** [comment]
> Can you provide the original file? Or a fragment of the original file (unzip the epub file and send me some xml/htm files)?

Yes, the translation does not work on the title and bulleted list in this example:

<h3 class="h3"><a id="ch01lev1"/>YES, THIS BOOK IS FOR YOU</h3>
<p class="noindent">A recent study showed that <a id="ch00fn11"/>only 12 percent of Americans are metabolically healthy, which means that only 12 percent of Americans have a perfectly functioning body—including healthy glucose levels. Odds are that <i>you</i>, and nine out of the ten people closest to you, are on a glucose roller coaster without knowing it.</p>
<p class="indent">Here are some questions to ask yourself to find out if your glucose levels are dysregulated.</p>
<ul class="blist">
<li class="bull">Have you been told by a doctor that you need to lose weight?</li>
<li class="bull">Are you trying to lose weight but finding it difficult?</li>
<li class="bull">Is your waist size (or pant size) above 40 inches if you are a man or above 35 inches if you are a woman? (<a id="ch00fn12"/>Waist size is better for predicting underlying disease than BMI is.)</li>
<li class="bull">Do you have extreme hunger pangs during the day?</li>
<li class="bull">Do you feel agitated or angry when you are hungry, aka <i>hangry</i>?</li>
<li class="bull">Do you need to eat every few hours?</li>
<li class="bull"><span aria-label="page 5" id="page_5" role="doc-pagebreak"/>Do you feel shaky, lightheaded, or dizzy if meals are delayed?</li>
<li class="bull">Do you crave sweet things?</li>
<li class="bull">Do you feel sleepy midmorning or midafternoon or are you tired all the time?</li>
<li class="bull">Do you need caffeine to keep you going throughout the day?</li>
<li class="bull">Do you have trouble sleeping or wake up with heart palpitations?</li>
<li class="bull">Do you have energy crashes where you break out in a sweat or get nauseous?</li>
<li class="bull">Do you suffer from acne, inflammation, or other skin conditions?</li>
<li class="bull">Do you experience anxiety, depression, or mood disorders?</li>
<li class="bull">Do you experience brain fog?</li>
<li class="bull">Is your mood variable?</li>
<li class="bull">Do you frequently get colds?</li>
<li class="bull">Do you experience acid reflux or gastritis?</li>
<li class="bull">Do you have hormonal imbalances, missed periods, PMS, infertility, or PCOS?</li>
<li class="bull">Have you ever been told that your glucose levels are elevated?</li>
<li class="bull">Do you have insulin resistance?</li>
<li class="bull">Do you have prediabetes or type 2 diabetes?</li>
<li class="bull">Do you have nonalcoholic fatty liver disease?</li>
<li class="bull">Do you have heart disease?</li>
<li class="bull">Do you have difficulty managing gestational diabetes?</li>
<li class="bull">Do you have difficulty managing type 1 diabetes?</li>
</ul>
<p class="indent">And most important: Do you think you could feel better than you currently do? If the answer is yes, keep reading.</p>

**Comment by Legi0nP** [comment]
> Can you provide the original file? Or a fragment of the original file (unzip the epub file and send me some xml/htm files)?

Yes, the translation does not work on the title and bulleted list the fragments for exemple: https://pastes.io/mzttutrm7m or https://pastebin.com/7kvF6fDB

`
<h3 class="h3"><a id="ch01lev1"/>YES, THIS BOOK IS FOR YOU</h3>
<p class="noindent">A recent study showed that <a id="ch00fn11"/>only 12 percent of Americans are metabolically healthy, which means that only 12 percent of Americans have a perfectly functioning body—including healthy glucose levels. Odds are that <i>you</i>, and nine out of the ten people closest to you, are on a glucose roller coaster without knowing it.</p>
<p class="indent">Here are some questions to ask yourself to find out if your glucose levels are dysregulated.</p>
<ul class="blist">
<li class="bull">Have you been told by a doctor that you need to lose weight?</li>
<li class="bull">Are you trying to lose weight but finding it difficult?</li>
<li class="bull">Is your waist size (or pant size) above 40 inches if you are a man or above 35 inches if you are a woman? (<a id="ch00fn12"/>Waist size is better for predicting underlying disease than BMI is.)</li>
<li class="bull">Do you have extreme hunger pangs during the day?</li>
<li class="bull">Do you feel agitated or angry when you are hungry, aka <i>hangry</i>?</li>
<li class="bull">Do you need to eat every few hours?</li>
<li class="bull"><span aria-label="page 5" id="page_5" role="doc-pagebreak"/>Do you feel shaky, lightheaded, or dizzy if meals are delayed?</li>
<li class="bull">Do you crave sweet things?</li>
<li class="bull">Do you feel sleepy midmorning or midafternoon or are you tired all the time?</li>
<li class="bull">Do you need caffeine to keep you going throughout the day?</li>
<li class="bull">Do you have trouble sleeping or wake up with heart palpitations?</li>
<li class="bull">Do you have energy crashes where you break out in a sweat or get nauseous?</li>
<li class="bull">Do you suffer from acne, inflammation, or other skin conditions?</li>
<li class="bull">Do you experience anxiety, depression, or mood disorders?</li>
<li class="bull">Do you experience brain fog?</li>
<li class="bull">Is your mood variable?</li>
<li class="bull">Do you frequently get colds?</li>
<li class="bull">Do you experience acid reflux or gastritis?</li>
<li class="bull">Do you have hormonal imbalances, missed periods, PMS, infertility, or PCOS?</li>
<li class="bull">Have you ever been told that your glucose levels are elevated?</li>
<li class="bull">Do you have insulin resistance?</li>
<li class="bull">Do you have prediabetes or type 2 diabetes?</li>
<li class="bull">Do you have nonalcoholic fatty liver disease?</li>
<li class="bull">Do you have heart disease?</li>
<li class="bull">Do you have difficulty managing gestational diabetes?</li>
<li class="bull">Do you have difficulty managing type 1 diabetes?</li>
</ul>
<p class="indent">And most important: Do you think you could feel better than you currently do? If the answer is yes, keep reading.</p>`
